main.go

```diff
@@ -15,12 +15,20 @@ import (
 	"github.com/giantswarm/kubectl-gs/pkg/errorprinter"
 )
 
+var (
+	DebugMode = false
+)
+
 func main() {
 	err := mainE(context.Background())
 	if err != nil {
-		ep := errorprinter.New()
-		fmt.Print(ep.Format(err))
-		os.Exit(1)
+		if DebugMode {
```

Comment:
Passing this pointer up is really ugly. How about doing `if containsString(args, "--debug")` and adding a flag only for `-h` output with a comment in code that it's ignored and handled in `main.go`?